beginner_source/export_simple_model_to_onnx_tutorial.py

```diff
@@ -0,0 +1,162 @@
+# Netron can either be installed on MacOS, Linux or Windows computers, or run directly from the browser.
+# Let's try the web version by opening the following link: https://netron.app/.
+#
+# .. image:: ../_static/img/onnx/netron_web_ui.png
+#
+# Once Netron is open, we can drag and drop our ``my_image_classifier.onnx`` file into the browser or select it after
+# clicking on `Open model` button.
+#
+# .. image:: ../_static/img/onnx/image_clossifier_onnx_modelon_netron_web_ui.png
+#
+# And that is it! We have successfully exported our PyTorch model to ONNX format and visualized it with Netron.
+#
+# The last step is executing the ONNX model with `ONNX Runtime`, but before we do that, let's install ONNX Runtime.
+# %%
+#  .. code-block:: bash
+#
+#      %%bash
+#      pip install onnxruntime
+
+# Adapt PyTorch input to ONNX format
+
+onnx_input = export_output.adapt_torch_inputs_to_onnx(torch_input)
+print(f"Input length: {len(onnx_input)}")
+print(f"Sample input: {onnx_input}")
+
+# in our example, the input is the same, but we can have more inputs
+# than the original PyTorch model in more complex cases.
+# Now we can execute the ONNX model with ONNX Runtime.
+
+import onnxruntime
+
+# We are using CPU as the execution provider, but ``providers=['CUDAExecutionProvider']`` enables CUDA too.
+
+ort_session = onnxruntime.InferenceSession("./my_image_classifier.onnx", providers=['CPUExecutionProvider'])
+
+# ONNX Runtime requires the input to be on CPU and using numpy Tensors,
+# so we need to convert our PyTorch input to numpy.
+
+def to_numpy(tensor):
+    return tensor.detach().cpu().numpy() if tensor.requires_grad else tensor.cpu().numpy()
+
+# ONNX Runtime also requires the input to be a dictionary with
+# the keys being the input name and the value the Numpy tensor
+
+onnxruntime_input = {k.name: to_numpy(v) for k, v in zip(ort_session.get_inputs(), onnx_input)}
+
+# Finally, we can execute the ONNX model with ONNX Runtime.
+
+onnxruntime_outputs = ort_session.run(None, onnxruntime_input)
+
+# The output can be a single tensor or a list of tensors, depending on the model.
+# Let's execute the PyTorch model and use it as benchmark next
+torch_outputs = torch_model(torch_input)
+
+# We need to adapt the PyTorch output format to match ONNX's
+torch_outputs = export_output.adapt_torch_outputs_to_onnx(torch_outputs)
+
+# Now we can compare both results
+assert len(torch_outputs) == len(onnxruntime_outputs)
+for torch_output, onnxruntime_output in zip(torch_outputs, onnxruntime_outputs):
+    torch.testing.assert_close(torch_output, torch.tensor(onnxruntime_output))
+
+print("PyTorch and ONNX Runtime output matched!")
+print(f"Output length: {len(onnxruntime_outputs)}")
+print(f"Sample output: {onnxruntime_outputs}")
+
+# That is about it! We have successfully exported our PyTorch model to ONNX format,
+# saved it to disk, executed it with ONNX Runtime and compared its result with PyTorch's.
```

Comment:
Here you can add links to other related tutorials/topics. Something like this:

Now that you have learned the basics of ONNx, you might also want to checkout the following tutorials:

* Link1
* Link2
* Link3

Reply:
We are planning to have one tutorial merged for now, as a poc.

In the coming weeks a series of tutorials will be worked on and we will be more than glad to list them here :)